### PR TITLE
Add FAQ for adding documentation

### LINKED ISSUE #1834: Document how users in the community can update documentations.
e.g.: what branch to use.

### PR DESCRIPTION
**Pull request by blueandgold**
Fixes #1834

### CODE REVIEW

**Comment by EricBeach** [comment]
Just a thought: Is it worth linking to the process for building the website locally so people can preview the updated documentation?

**Comment by blueandgold** [comment]
Done

File: _faq/general/updating-documentation.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+order: 2
+---
+{::options auto_ids="false" /}
+
+Forseti documentations can be updated in the `forsetisecurity.org-dev` branch.
+
+You can test your documentation changes locally, by
+[following the instructions here]({% link _docs/latest/develop/website.md %}).
+
+After your updated documentation is merged, the Forseti team will merge it
```
Comment (joecheuk) [inline _faq/general/updating-documentation.md]: Should we specify the timeline of that happening, maybe per release?

Comment (blueandgold) [reply _faq/general/updating-documentation.md]: Done

**Pull request merged**